harness/determined/common/api/bindings.py

```diff
@@ -4642,6 +4642,7 @@ class v1GetUsersRequestSortBy(enum.Enum):
     SORT_BY_ADMIN = "SORT_BY_ADMIN"
     SORT_BY_ACTIVE = "SORT_BY_ACTIVE"
     SORT_BY_MODIFIED_TIME = "SORT_BY_MODIFIED_TIME"
+    SORT_BY_NAME = "SORT_BY_NAME"
```

Comment:
What does sort_by_name mean here? I guess it is group name because username and display name are already options?

Reply:
oh I'm seeing the COALESCE query now